File: rohitMigrationTesting/pipelines/pipeline2/code/src/main/scala/simpledatalabs_27/rohitmigrationtesting/pipeline/pipeline2/graph/DS1.scala
package simpledatalabs_27.rohitmigrationtesting.pipeline.pipeline2.graph

import io.prophecy.libs._
import simpledatalabs_27.rohitmigrationtesting.pipeline.pipeline2.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object DS1 {

  def apply(context: Context): DataFrame =
    context.spark.read
      .format("csv")
      .option("header",                                   true)
      .option("sep",                                      ",")
      .schema(StructType(Array(StructField("customer_id", StringType, true))))
      .load("dede")

}

File: plibsTest/pipelines/p1/code/p1/graph/ds1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.utils import *
from prophecy.libs import typed_lit
from p1.config.ConfigStore import *
from p1.udfs.UDFs import *

def ds1(spark: SparkSession) -> DataFrame:
    return spark.read\
        .schema(StructType([StructField("id", StringType(), True)]))\
        .option("header", True)\
        .option("sep", ",")\
        .csv("dede")
